src/__tests__/VagaConsultas.test.ts

```diff
@@ -0,0 +1,79 @@
+import request from 'supertest';
+import { Connection, getConnection } from 'typeorm';
+import { app } from '../app';
+import { v4 as uuid } from "uuid";
+
+import createConnection from '../database';
+let connection: Connection;
+
+describe("vagasConsultas", () => {
+    beforeAll(async () => {
+        connection = await createConnection();
+        await connection.runMigrations();
+    });
+
+    beforeEach(async () => {
+        connection.undoLastMigration();
+        await connection.close();
+        connection = await createConnection();
+        await connection.runMigrations();
+    })
```

Comment:
porque usar beforeEach nesses testes?

Reply:
Eu coloquei pra tentar tratar a lentidão na finalização dos testes, mas não resolveu. Posso tirar quando for ajustar o **VagaConsultasController.ts**